src/main/java/doit/jpastudy2/JpaRepository/Champion.java

```diff
@@ -0,0 +1,28 @@
+package doit.jpastudy2.JpaRepository;
+
+import jakarta.persistence.*;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@NoArgsConstructor
+@Getter
+public class Champion {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "Id_champion")
+    private Long id;
```

Comment:
익숙한 주제로 작성하는 것 매우 좋습니다ㅎㅎㅎ
틀린 것은 아니지만! 관례 상 자주 사용되는 패턴이 있습니다.
MySQL에서는 Column 이름을 snake_case로 작성하는 것이 일반적입니다. [참고!](https://yummy0102.tistory.com/508)
그리고 table명_id 구조로 PK를 작성하는 편이기도 합니다. 
따라서 아래처럼 써보시는 것이 어떠할까 싶네용

```suggestion
    @Column(name = "champion_id")
    private Long id;
```